src/Widgets/TestimonialWidget/TestimonialWidgetEditingConfig.js

```diff
@@ -20,6 +20,20 @@ Scrivito.provideEditingConfig("TestimonialWidget", {
     author: "Someone",
     testimonial: "This is great!",
   },
+  validations: [
+    [
+      "author",
+
+      author => {
+        if (!author) {
+          return {
+            message: "The author should be set.",
+            severity: "warning",
+          };
+        }
+      },
+    ],
+  ],
   titleForContent: widget =>
     `${widget.get("author")}: ${truncate(widget.get("testimonial"))}`,
```

Comment:
With the new validation `titleForContent` looks strange now:

![Testimonial](https://user-images.githubusercontent.com/86275/72075579-58b6b780-32f4-11ea-854d-f91fe18fe74d.png)

It's probably best to simply delete `titleForContent` on the Testimonial Widget or adjust it, so that it contains "Testimonial" in it (e.g. "Testimonial by XY" or something).

```suggestion
    `Testimonial: ${widget.get("author")} - ${truncate(widget.get("testimonial"))}`,
```